### PR TITLE
Capstone mobile version

### PR DESCRIPTION
**Pull request by Outkarm**
- add all images for the home and about pages
- add home page HTML with respect to the Creative Commons structure by Cindy Shin in Behance
- add about page HTML with respect to the Creative Commons structure by Cindy Shin in Behance
- add style to the home and about page
- add javascript to the home and about page to load the featured speakers dynamically 
- add ReadMe
- add a social bar on top of the nav bar
- add hover to program cards
- add icon images to the social bar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        Magazines. Though there was a brief experiment in bringing back
+        superheroes such as Captain America in 1953, Atlas’s output was mostly
+        in other genres such as humour, westerns, horror, war, and science
+        fiction.
+      </p>
+      <div class="time-line">2015.10.15(THU) ~ 16(FRI)</div>
+      <div class="location">
+        @ Royal Victoria Dock 1 Western Gateway London E16 1XL London, UK
+      </div>
+    </section>
```
Comment (mukhammadrizooff) [inline index.html:106]: - [ ] [REQUIRED] Good job so far! You forgot to make the elements shown at the `top` of the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) on the main page. Please create it!

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+          </div>
+          <h3 class="program-title">Growth</h3>
+          <p class="program-info">
+            Be the change you want to see in this world.
+          </p>
+        </div>
+      </div>
+      <button class="join-program">Join Marvel Divide 2015</button>
+      <a href="" class="see-program">SEE THE WHOLE PROGRAME</a>
+    </section>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! The `top` and `bottom` of the `<p/>` elements in the program section should be given the same amount of `padding` or `margin` as in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), and don't forget to give your cards a `hover effect` as shown in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)!

image

File: README.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+- HTML5, CSS3, JavaScript
+- Linters:
+  - Stylelint
+  - Lighthouse
+  - Webhint
+  - ESLint
+
+## Project Documentation
+
+- Here is the [link]() to my loom video
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED] You should record a `video`(`Video presentation`) presenting the features of the project!

image

File: README.md [hunk 4]
```diff
@@ -1 +1,66 @@
...
+
+## Website Mockup 📱 💻 🖥️
+
+![mobile-look](./Images/mobile.gif)
+![Laptop-look](./Images/laptop.gif)
+
+## Live Demo
+
+[Capstone I : Marvel Divide 2015]()
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [OPTIONAL] It would be better if you could deploy your project 😃

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Outkarm, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 1 above)
Comment (Outkarm) [reply index.html:106]: thank you for the correction 👍

File: index.html (hunk 2 above)
Comment (Outkarm) [reply index.html]: thanks for the tip

File: README.md (hunk 3 above)
Comment (Outkarm) [reply README.md]: good  looking out

File: README.md (hunk 4 above)
Comment (Outkarm) [reply README.md]: would do that

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Outkarm       :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**